web/public/locales/ko.json

```diff
@@ -113,17 +113,17 @@
     "legend": "범주",
     "faq": "자주 묻는 질문"
   },
-  "wind": "바람",
-  "solar": "태양",
-  "hydro": "전기",
-  "hydro storage": "전기 저장",
-  "battery storage": "배터리 용량",
+  "wind": "풍력",
+  "solar": "태양광",
+  "hydro": "수력",
+  "hydro storage": "양수",
```

Comment:
Right now this says something like pumping if I got my translations right?

Should it not say hydro storage?